### ISSUE TITLE
Error: Unexpected key 'moduleName' found, expected one of: acorn, amd, banner, cache,...

### ISSUE THREAD

**Issue by playground** [jvandemo/generator-angular2-library#195]
Recent rollup update is breaking the bundling.
```
[14:18:07] Finished 'clean:build' after 12 ms
[14:18:07] Starting 'clean'...
[14:18:07] Finished 'clean' after 60 μs
[14:18:07] Starting 'build'...
[14:18:07] Finished 'build' after 1.18 μs
[14:18:07] Finished 'copy:source' after 31 ms
[14:18:07] Starting 'inline-resources'...
[14:18:07] Finished 'inline-resources' after 8.45 ms
[14:18:07] Starting 'ngc'...
[14:18:08] Finished 'ngc' after 983 ms
[14:18:08] Starting 'rollup:fesm'...
options.entry is deprecated, use options.input
[14:18:08] Finished 'rollup:fesm' after 137 ms
[14:18:08] Starting 'rollup:umd'...
options.entry is deprecated, use options.input

events.js:163
      throw er; // Unhandled 'error' event
      ^
Error: Unexpected key 'moduleName' found, expected one of: acorn, amd, banner, cache, context, entry, exports, extend, external, file, footer, format, globals, indent, input, interop, intro, legacy, moduleContext, name, noConflict, onwarn, output, outro, paths, plugins, preferConst, pureExternalModules, sourcemap, sourcemapFile, strict, targets, treeshake, watch

```

**Comment by playground** [comment]
Found the documentation to fix the errors.

Need to change the following:
input -> entry
moduleName -> name

**Comment by jvandemo** [comment]
Thank you, version 11.4.0 has been released with a fix (see #190) 👍
